nx-dev/data-access-ai/src/lib/data-access-ai.ts

```diff
@@ -72,18 +61,26 @@ export async function queryAi(
   }
 
   try {
-    checkEnvVariables(openAiKey, supabaseUrl, supabaseServiceKey);
+    checkEnvVariables(supabaseUrl, supabaseServiceKey);
 
     if (!query) {
       throw new UserError('Missing query in request data');
     }
 
     // Moderate the content to comply with OpenAI T&C
     const sanitizedQuery = query.trim();
-    const moderationResponse: CreateModerationResponse = await openai
-      .createModeration({ input: sanitizedQuery })
-      .then((res) => res.data);
+    const moderationResponseObj = await fetch('/api/openai-handler', {
```

Comment:
These API calls could be in a helper function that you call here it's small nit-pick though

Reply:
Done! :D